client/src/pages/home.js

```diff
@@ -1,19 +1,36 @@
-import React from "react";
+import React, { Component } from "react";
+import Main from "./Main";
 
-const Home = () => {
-    return(
-        <div className={'container-fluid'}>
-            <section className="home" id="home">
-            <h1> Welcome to the Graduate Assistant Systems Scheduler (GASS)!</h1>
-            <p> Here we aim to solve your problems of scheduling conflicts between
-                busy schedules of Graduate Assistants and Teaching Assistants as well
-                as meeting the needs of faculty and course requirements. This software also
-                schedules the necessary time required for lab meetings while respecting the time
-                constraints of half and full-time graduate/teaching assistants.
-            </p>
-            </section>
-        </div>
-    )
+class Home extends Component {
```

Comment:
Home page which should be the first page the user sees upon opening the web form.